src/main/java/javax/servlet/AsyncContext.java

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (c) 2017, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2017, 2018 Oracle and/or its affiliates and others.
+ * All rights reserved.
```

Comment:
Typically I'm cautious about reformatting such copyright lines, as they often are targets of tools to update etc.
On what basis have these lines been split? There are longer lines elsewhere in comments?